openshift-install-powervs

```diff
@@ -481,15 +481,14 @@ function verify_data {
       error "File pull-secret.txt not found"
     fi
   fi
-  if [ -f "./$ARTIFACTS_DIR/data/id_rsa" ] && [ -f "./$ARTIFACTS_DIR/data/id_rsa.pub" ]; then
-    log "Found id_rsa & id_rsa.pub in data directory"
-  elif [ -f "./id_rsa" ] && [ -f "./id_rsa.pub" ]; then
-    log "Found id_rsa & id_rsa.pub in current directory"
-    cp -f ./id_rsa ./id_rsa.pub ./"$ARTIFACTS_DIR"/data/
-  else
-    warn "Creating new SSH key-pair..."
-    ssh-keygen -t rsa -f ./id_rsa -N ''
-    cp -f "./id_rsa" "./id_rsa.pub" ./"$ARTIFACTS_DIR"/data/
+  if [ ! -s "$VAR_PRIVATE_KEY_FILE" ] && [ ! -s "$VAR_PUBLIC_KEY_FILE" ]; then
+    if [ -s "./id_rsa" ] && [ -s "./id_rsa.pub" ]; then
+      cp -f ./id_rsa ./id_rsa.pub ./"$ARTIFACTS_DIR"/data/
```

Comment:
Can you keep the log statement here.. this will be when user provides all variables except the key files right?

Reply:
yes